meta-client/src/main/java/org/apache/helix/metaclient/api/MetaClientInterface.java

```diff
@@ -202,6 +202,14 @@ public Stat (EntryMode mode, int version, long ctime, long mtime, long etime) {
    */
   T get(final String key);
 
+  /**
+   * Fetch the data for a given key.
+   * @param key key to identify the entry
+   * @param returnNullIfPathNotExists throws MetaClientNoNodeException if no node when set to false;
+   * @return Return data of the entry.
+   */
+  T get(String key, boolean returnNullIfPathNotExists);
```

Comment:
Why we add this API? Was the original get not return NULL?

Reply:
+1. The above "get" method states it will return null if key doesn't exist.